### DISCUSSION TITLE
Is it possible to show the `:` when using Cmdline

### DISCUSSION THREAD

**Discussion by felipesere** [folke/noice.nvim#868] [Q&A] [answered]
Its minor and cosmetic, but I was wondering if I can have the `:` back when I type `:wq` and the really neat Cmdline popup shows?

**Comment by folke** [comment]
Check the docs??

**Comment by ilias777** [comment, marked as answer]
You can read &rarr; [here](https://github.com/folke/noice.nvim/blob/main/doc/noice.nvim.txt#L128) &larr; the docs how you can configure the cmdline icon.

Set this in your noice configuration:

```lua
opts = {
    cmdline = {
        format = {
            cmdline = { pattern = '^:', icon = ':', lang = 'vim' },
        },
    },
},
```
<img width="1330" alt="cmdline-icon" src="https://github.com/folke/noice.nvim/assets/39212564/c6b632e8-1200-4803-b264-abe07fe439cc">
